chromium_src/chrome/browser/ui/views/tabs/tab_style_views.cc

```diff
@@ -31,8 +33,21 @@ class BraveGM2TabStyle : public GM2TabStyle {
   TabStyle::TabColors CalculateColors() const override;
   const gfx::FontList& GetFontList() const override;
   SeparatorBounds GetSeparatorBounds(float scale) const override;
+  void PaintTab(gfx::Canvas* canvas) const override;
 
  private:
+  bool IsVerticalTab() const {
+    if (!base::FeatureList::IsEnabled(tabs::features::kBraveVerticalTabs))
+      return false;
+
+    return tabs::features::ShouldShowVerticalTabs(
+        tab_->controller()->GetBrowser());
+  }
+
+  bool IsInactiveAndInGroup() const {
```

Comment:
nit: `IsInGroupAndNotActive` be better imo to avoid reading it as "is in active" as in "is in active state"